src/network/httpd.py

```diff
@@ -1,28 +1,34 @@
+"""
+src/network/httpd.py
+=======================
+"""
 import asyncore
 import socket
 
 from tls import TLSHandshake
 
+
 class HTTPRequestHandler(asyncore.dispatcher):
+    """Handling HTTP request"""
     response = """HTTP/1.0 200 OK\r
-Date: Sun, 23 Oct 2016 18:02:00 GMT\r
-Content-Type: text/html; charset=UTF-8\r
-Content-Encoding: UTF-8\r
-Content-Length: 136\r
-Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r
-Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r
-ETag: "3f80f-1b6-3e1cb03b"\r
-Accept-Ranges: bytes\r
-Connection: close\r
-\r
-<html>
-<head>
-  <title>An Example Page</title>
-</head>
-  <body>
-     Hello World, this is a very simple HTML document.
-  </body>
-</html>"""
+    Date: Sun, 23 Oct 2016 18:02:00 GMT\r
```

Comment:
I think this will break the protocol, but the file is obsolete, I only used it for testing while developing, so it doesn't matter.